auth.proto

```diff
@@ -14,13 +14,18 @@ service AuthzService {
   rpc Authorize(AuthorizeReq) returns (AuthorizeRes) {}
 }
```

Comment:
Instead of `AuthzService` can we have like `ThingsAuthzService` ? 
```suggestion
service ThingsAuthzService {
  // Authorize checks if the subject is authorized to perform
  // the action on the object.
  rpc Authorize(AuthorizeReq) returns (AuthorizeRes) {}
}
```

Reply:
Okay.